README.md

```diff
@@ -31,7 +32,7 @@ Include the following apple_store_source package version in your `packages.yml`
 ```yaml
 packages:
   - package: fivetran/apple_store_source
-    version: [">=0.4.0", "<0.5.0"] # we recommend using ranges to capture non-breaking changes automatically
+    version: 0.5.0-a1
 ```
 ### Step 3: Define database and schema variables
 By default, this package runs using your destination and the `apple_store` schema. If this is not where your apple_store data is (for example, if your apple_store schema is named `apple_store_fivetran`), add the following configuration to your root `dbt_project.yml` file:
```

Comment:
Let's also update this based on the previous comment to state the following
```suggestion
By default, this package runs using your destination and your Apple Store schema (default `itunes_connect`). If this is not where your Apple Store data is (for example, if your Apple Store schema is named `apple_store_fivetran`), add the following configuration to your root `dbt_project.yml` file:
```

Reply:
updated